user/pages/03.commerce2/02.developer-guide/03.products/01.overview/02.concepts/docs.md

```diff
@@ -4,8 +4,54 @@ taxonomy:
     category: docs
 ---
 
-What are these things:
-1. Product types, Default product type, Products
-2. Variation types, Default product variation type, Variations, default order item type, Sku uniqueness
-3. Product attributes, attribute values
-4. Purchasable entities
+This page describes Drupal Commerce products at a conceptual level. For step-by-step instructions for setting up products for your site, see the [Product Architecture documentation](../../02.product-documentation). For more specific, technical explanations of the product-related data structures and relationships, see the [Product Information Structure documentation](../03.product-information-structure).
+
+#### Product SKUs -- What are they and why are they important?
+SKU stands for "stock keeping unit" and is a unique code that identifies the products you sell. Companies that are direct manufacturers may use MPNs, or "manufacturing part numbers", instead of SKUs (but they'll still be called "SKUs" in Drupal).
+
+For stores that sell physical products, SKUs are critically important for inventory management. A complete list of all product SKUs along with a quantity value for each provides the information you need to monitor stock levels. What about stores that sell non-physical products, like services or virtual products, for which there may be *unlimited* inventory? You still need SKUs. SKUs are important for sales and financial reporting for all types of products.
```

Comment:
I would think of maybe just adding a quick "The SKU is your unique identifier for that product", to help drive the point "it's a unique identifier."